src/mat/4C_mat_micromaterial.hpp

```diff
@@ -43,7 +51,9 @@ namespace Mat
       ///
       const double initvol_;
 
-      //@}
+      //// runtime output option
+      runtime_output_option RuntimeOutputOption;
```

Comment:
Does this confirm with our style guides? Shouldn't this be stake case?

Reply:
changed this according to the style guide

Reply:
/// should be enough?